spec/helpers/lib/space.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+module.exports = (fsAsync, logger, path, unique) => {
+    /**
+     * This class exists to organize all artifacts within a
+     * certain test. For example, if I did a concurrency test
+     * where I uploaded many artifacts (or uploaded many metadatas)
+     * they would all be contained within the same "testPath".
+     */
+    class Space {
+        /**
+         * testPath is the path to a directory that this Space
```

Comment:
I had a discussion with Tyler about this. He said if it is long he puts it in the description. Should we talk to him again?